semgrep-core/src/parsing/other/yaml_to_generic.mli

```diff
@@ -1,10 +1,11 @@
 (* Parsing a YAML file.
  * This may raise Parse_info.Other_error.
  *)
-val program : Common.filename -> AST_generic.program
+val program : is_target:bool -> Common.filename -> AST_generic.program
```

Comment:
I don't think you need a label. When we use the program function, it's a target,
when we use any, it's always to parse a pattern.